WcaOnRails/spec/controllers/persons_controller_spec.rb

```diff
@@ -21,7 +21,7 @@
         json_person = json['rows'][0]
         expect(json_person['name']).to include "Jennifer Lawrence"
         expect(json_person['wca_id']).to eq "2016LAWR01"
-        expect(json_person['country']).to eq "USA"
+        expect(json_person['country']).to eq "United States"
```

Comment:
I don't mind the change. As @larpetrus has menioned in the past, I kind of like using somebody else's data for this sort of stuff, as it leaves us apolitical.